test/versioned/mysql2/basic-pool.tap.js

```diff
@@ -379,6 +345,40 @@ tap.test('mysql2 built-in connection pools', function (t) {
       })
     })
   })
+
```

Comment:
why move this? is it just to make it so it's the last one since it's optional depending on where we're running?

Reply:
NVM, I see now, this is to put it in the same place as in the mysql2 tests, right?